app/kuma-ui/pkg/server/server.go

```diff
@@ -64,6 +76,21 @@ func (g *Server) Start(stop <-chan struct{}) error {
 	}
 }
 
+func (g *Server) apiHandler() (http.Handler, error) {
+	apiServerUrl, err := url.Parse(fmt.Sprintf("http://localhost:%d", g.ApiServerPort))
```

Comment:
Why do we need to hardcode it here instead of making it a part of configuration ?

Reply:
Moved as a part of the configuration.